### PR TITLE
[ADF-3912] Document-List is not able to retrieve the -file-plan- information from the node

### PR DESCRIPTION
**Pull request by abhi4**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by eromano** [comment]
@abhi4 can you please sign the CLA? thanks

File: lib/core/app-config/schema.json [hunk 1]
```diff
@@ -623,7 +623,8 @@
               "-favorites-",
               "-recent-",
               "-sharedlinks-",
-              "default"
+              "default",
```
Comment (eromano) [inline lib/core/app-config/schema.json]: can you please invert those two lines and keep default as last?

File: lib/content-services/document-list/services/custom-resources.service.ts [hunk 2]
```diff
@@ -212,6 +212,37 @@ export class CustomResourcesService {
         }).pipe(catchError((err) => this.handleError(err)));
     }
 
+    /**
+     * Gets all record categories in the file plan.
+     * @param pagination Specifies how to paginate the results
+     * @returns List of record categories
+     */
+    loadFilePlan(nodeId: string, pagination: PaginationModel): Observable<NodePaging> {
```
Comment (eromano) [inline lib/content-services/document-list/services/custom-resources.service.ts]: Not sure it return a NodePaging, can you check the return type?

File: lib/content-services/document-list/services/custom-resources.service.ts [hunk 3]
```diff
@@ -212,6 +212,37 @@ export class CustomResourcesService {
...
+            include: ['properties', 'aspectNames'],
+            maxItems: pagination.maxItems,
+            skipCount: pagination.skipCount
+        };
+
+        return new Observable((observer) => {
+            this.apiService.getInstance().gsCore.fileplansApi.getFilePlanCategories(nodeId, options)
+                .then((page: any) => {
+                        page.list.entries.map(
+                            ({ entry }: any) => {
```
Comment (eromano) [inline lib/content-services/document-list/services/custom-resources.service.ts]: is this entry any type?

File: lib/content-services/document-list/services/custom-resources.service.ts [hunk 4]
```diff
@@ -212,6 +212,37 @@ export class CustomResourcesService {
...
+    loadFilePlan(nodeId: string, pagination: PaginationModel): Observable<NodePaging> {
+        const options = {
+            include: ['properties', 'aspectNames'],
+            maxItems: pagination.maxItems,
+            skipCount: pagination.skipCount
+        };
+
+        return new Observable((observer) => {
+            this.apiService.getInstance().gsCore.fileplansApi.getFilePlanCategories(nodeId, options)
+                .then((page: any) => {
```
Comment (eromano) [inline lib/content-services/document-list/services/custom-resources.service.ts]: is this page an any type?

File: lib/content-services/document-list/services/custom-resources.service.ts [hunk 5]
```diff
@@ -212,6 +212,37 @@ export class CustomResourcesService {
         }).pipe(catchError((err) => this.handleError(err)));
     }
 
+    /**
+     * Gets all record categories in the file plan.
+     * @param pagination Specifies how to paginate the results
```
Comment (eromano) [inline lib/content-services/document-list/services/custom-resources.service.ts]: this parameter doc is incomplete

**Comment by abhi4** [comment]
closing this pr because there are other changes that need to be done for the particular issue.

**Pull request closed without merging**